components-core/src/main/java/com/adyen/checkout/components/core/PaymentMethod.kt

```diff
@@ -20,18 +21,25 @@ import kotlinx.parcelize.Parcelize
 import org.json.JSONException
 import org.json.JSONObject
 
+@OptIn(IgnoredCustomizedField::class)
 @Parcelize
 data class PaymentMethod(
-    var type: String? = null,
-    var name: String? = null,
-    var brands: List<String>? = null,
-    var brand: String? = null,
-    var fundingSource: String? = null,
-    var issuers: List<Issuer>? = null,
-    var configuration: Configuration? = null,
-    var details: List<InputDetail>? = null,
+    val type: String? = null,
```

Comment:
Changing `var` to `val` is sort of a breaking change 🤔 I'm all for it, but I'm not sure if we should do it now

Reply:
I see, perhaps merchants manipulate these fields to get custom behavior in their implementations?

Reply:
Yes indeed, let's discuss it with the rest of the team